opencart3.0.x/upload/admin/controller/extension/payment/xenditcc.php

```diff
@@ -68,13 +68,11 @@ public function install() {
     }
 
     public function uninstall() {
-        if ($this->user->hasPermission('modify', 'extension/extension')) {
-            $this->load->model('extension/payment/xenditcc');
-            $this->model_extension_payment_xenditcc->uninstall();
-        }
+        $this->load->model('extension/payment/xendit');
+        $this->model_extension_payment_xendit->removePermission('cc');
```

Comment:
@andykim Curious why this block is different from the other payment method? It's not using `self::XENDIT_CODE` Is it because it's specific to `cc`?

Reply:
@ikanyu updated using `XENDIT_CODE=cc`